### PR TITLE
Add example of Loggers

### PR DESCRIPTION
**Pull request by nohwnd**

### CODE REVIEW

**Comment by PRMerger20** [comment]
@nohwnd : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Mikejo5000** [comment]
@pavelhorak @AbhitejJohn Please take a look. Thanks!

File: docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md [hunk 1]
```diff
@@ -305,6 +326,32 @@ public void HomePageTest()
...
+<LoggerRunSettings>
+    <Loggers>        
+      <Logger friendlyName="console" enabled="True">
+        <Configuration>
+            <Verbosity>quiet</Verbosity>
+        </Configuration>
+      </Logger>
+      <Logger friendlyName="trx" enabled="True">
+        <Configuration>
+          <LogFileName>foo.trx</LogFileName>
```
Comment (AbhitejJohn) [inline docs/test/configure-unit-tests-by-using-a-dot-runsettings-file.md:341]: Can we also add where this file will be output to?

**Comment by Mikejo5000** [comment]
@nohwnd Jakub, looks like this got merged. Would you mind opening a new PR to address comment from Abhitej? Thx!

**Comment by ktoliver** [comment]
@Mikejo5000 @AbhitejJohn Apologies--I didn't observe that approval of the commit conflicted with a request for a revision (and so, a need for clarification or explicit approval). Sorry for the oversight.

**Pull request merged**